bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/api/CatalogQuery.scala

```diff
@@ -17,27 +17,35 @@ case class MagnitudeQueryFilter(mc: MagnitudeConstraints) extends QueryResultsFi
   def filter(t: SiderealTarget): Boolean = mc.filter(t)
 }
 
-sealed abstract class CatalogName(val id: String, val displayName: String) {
+sealed abstract class CatalogName(val id: String, val displayName: String) extends Product with Serializable {
   def supportedBands: List[MagnitudeBand] = Nil
   // Indicates what is the band used when a generic R band is required
   def rBand: MagnitudeBand = MagnitudeBand.UC
 }
 
-case object SDSS extends CatalogName("sdss9", "SDSS9 @ Gemini")
-case object GSC234 extends CatalogName("gsc234", "GSC234 @ Gemini")
-case object PPMXL extends CatalogName("ppmxl", "PPMXL @ Gemini") {
-  override val supportedBands = List(MagnitudeBand.B, MagnitudeBand.R, MagnitudeBand.I, MagnitudeBand.J, MagnitudeBand.H, MagnitudeBand.K)
-  override val rBand: MagnitudeBand = MagnitudeBand.R
-}
-case object UCAC4 extends CatalogName("ucac4", "UCAC4 @ Gemini") {
-  override val  supportedBands = List(MagnitudeBand._g, MagnitudeBand._r, MagnitudeBand._i, MagnitudeBand.B, MagnitudeBand.V, MagnitudeBand.UC, MagnitudeBand.J, MagnitudeBand.H, MagnitudeBand.K)
-}
-case object TWOMASS_PSC extends CatalogName("twomass_psc", "TwoMass PSC @ Gemini")
-case object TWOMASS_XSC extends CatalogName("twomass_xsc", "TwoMass XSC @ Gemini")
-case object SIMBAD extends CatalogName("simbad", "Simbad")
-
 object CatalogName {
-  implicit val equals: Equal[CatalogName] = Equal.equal[CatalogName]((a, b) => a.id === b.id)
+
+  case object SDSS extends CatalogName("sdss9", "SDSS9 @ Gemini")
```

Comment:
Maybe we should mention that these names are linked to `skycat.cfg` (Is it so?). Just in case somebody wants to rename them

Reply:
I think there is no longer a dependency on `skycat.cfg`. I found the URLs I was looking for in `VoTableClient`. I suppose all bets are off with the OT though. If I find out that there is some connection to `skycat.cfg` I'll deal with it then.